From: Richard B Winters <rik@mmogp.com>
Date: Fri, 17 Apr 2015 08:46:31 -0400
Subject: Fix spelling of output and improve grammar

The spelling of output as ouput within the source throws a lintian
warning. This patch resolves that.

The grammar was also a tad off, so it was improved.

Change-Id: I5ad23a1fb89c5fdb39ba1f9bb13905ff36de82ad
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 lrm/lrmd/lrmd.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lrm/lrmd/lrmd.c b/lrm/lrmd/lrmd.c
index 385096b..188a4c0 100644
--- a/lrm/lrmd/lrmd.c
+++ b/lrm/lrmd/lrmd.c
@@ -3362,9 +3362,9 @@ on_ra_proc_finished(ProcTrack* p, int status, int signo, int exitcode
 						, op->rapop);
 		} else {
 			lrmd_log(LOG_WARNING, "There is something wrong: the "
-				"first line isn't read in. Maybe the heartbeat "
-				"does not ouput string correctly for status "
-				"operation. Or the code (myself) is wrong.");
+				"first line isn't read in. Maybe heartbeat does "
+				"not ouptut the string correctly for the status "
+				"operation, or perhaps the code is wrong.");
 		}
 	}
 
